dpnp/dpnp_iface_types.py

```diff
@@ -59,8 +59,14 @@
     "inf",
     "int",
     "int_",
+    "int8",
+    "int16",
     "int32",
     "int64",
+    "uint8",
```

Comment:
Need to add `uint`, `uintc`, `unitp` also

Reply:
Added